### PR TITLE
Mecanum Wheel Odometry in firmware_message_converter

### PR DESCRIPTION
**Pull request by Bitterisland6**
Added handling of "/firmware/wheel_odom_mecanum" topic in firmware message converter script

### CODE REVIEW

File: leo_bringup/config/firmware.yaml
Comment (bjsowa) [inline leo_bringup/config/firmware.yaml:1]: This file does not belong to this PR, delete it

File: leo_fw/src/firmware_message_converter.cpp [hunk 1]
```diff
@@ -290,10 +350,15 @@ class FirmwareMessageConverter : public rclcpp::Node
...
   rclcpp::Subscription<leo_msgs::msg::WheelOdom>::SharedPtr wheel_odom_sub_;
+  rclcpp::Subscription<leo_msgs::msg::WheelOdomMecanum>::SharedPtr wheel_odom_mecanum_sub_;
   rclcpp::Subscription<leo_msgs::msg::Imu>::SharedPtr imu_sub_;
 
   // Service
   rclcpp::Service<leo_msgs::srv::SetImuCalibration>::SharedPtr set_imu_calibration_service;
+
+  // Flags
+  bool wheel_odom_advertised;
+  bool wheel_odom_mecanum_advertised;
```
Comment (bjsowa) [inline leo_fw/src/firmware_message_converter.cpp]: Use `_` prefix for private member variables

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: leo_bringup/config/firmware.yaml
Comment (Bitterisland6) [reply leo_bringup/config/firmware.yaml:1]: Done

File: leo_fw/src/firmware_message_converter.cpp (hunk 1 above)
Comment (Bitterisland6) [reply leo_fw/src/firmware_message_converter.cpp]: Removed those flags as I switched to different solution

**Review by bjsowa — APPROVED** [review]

**Pull request merged**